### PR TITLE
Mongodb adapter wy

### PR DESCRIPTION
**Pull request by Wynnfan**
Provide method can modify the table name.

### CODE REVIEW

File: adapter.go [hunk 1]
```diff
@@ -91,11 +104,40 @@ func (a *adapter) open() {
...
+		dI.Database = "casbin"
+	}
+
+	session, err := mgo.DialWithInfo(dI)
+	if err != nil {
+		panic(err)
+	}
+
+	db := session.DB(dI.Database)
+	collection := db.C(name)
```
Comment (hsluoyz) [inline adapter.go]: The ``name`` here should be ``"casbin_rule"``, because it's the table name, not the db name.

File: adapter.go [hunk 2]
```diff
@@ -61,6 +61,19 @@ func NewAdapter(url string) persist.Adapter {
 	return a
 }
 
+// Can use provided database name.
+func NewDBAdapter(url, name string) persist.Adapter {
```
Comment (hsluoyz) [inline adapter.go]: Please do not create another constructor method. Just do it as how we did it in Xorm Adapter: https://github.com/casbin/xorm-adapter/blob/6d86407ce2631ca1f6f6943374c6129c5b03e190/adapter.go#L50-L75

File: adapter.go (hunk 1 above)
Comment (Wynnfan) [reply adapter.go]: ok, the name really is the table name. I found write the wrong notes when i push,but i have modified,so have three commit.

Comment (hsluoyz) [reply adapter.go]: So you should squash your commits into one in this PR. Because there's no need to see how you fix the errors in the commit history.

Comment (Wynnfan) [reply adapter.go]: Thank you，I am a rookie. About this name modify, is it still necessary to submit？

Comment (hsluoyz) [reply adapter.go]: Please join the Tencent QQ group for discussion: 546057381

**Pull request closed without merging**